benchmarks/kernels/bench_fp8_gemm.py

```diff
@@ -197,21 +140,13 @@ def prepare_shapes(args):
         nargs="+",
         type=str,
         default=["meta-llama/Llama-3.1-8B-Instruct"],
-        choices=[*WEIGHT_SHAPES.keys()],
-        help="List of models to benchmark",
```

Comment:
why remove the help here?

Reply:
I think the name "models" already show the meaning and we can save some space for "help", but certainly I can add it back if you wish.